anesthetic/samples.py

```diff
@@ -196,6 +196,14 @@ def plot_1d(self, axes, *args, **kwargs):
         kwargs['kind'] = kwargs.get('kind', 'kde_1d')
         kwargs['label'] = kwargs.get('label', self.label)
 
+        # TODO: remove this in version >= 2.1
+        if kwargs['kind'] == 'kde':
+            warnings.warn(
+                "You are using \'kde\' as a plot kind. "
+                "\'kde_1d\' is the appropriate keyword for anesthetic. "
+                "Your plots may look odd if you use this argument."
+                )
```

Comment:
I believe the same should go for `lkwargs['kind'] == 'hist'`. So probably good to add that here, too.

Reply:
Done in 549979c